packages/2018/src/components/HomePage.js

```diff
@@ -697,10 +702,10 @@ class HomePage extends Component {
                   </p>
                 </div>
                 <div className={brainWrapper}>
-                  <img src={brain} />
+                  <img src={brain} alt="" />
```

Comment:
Not sure what the page looks like, so depending on that an alt may be nice here

Reply:
Yeah, these are decorative, so empty alt is intentional! Thanks for thinking about it!